setup.py

```diff
@@ -47,5 +47,7 @@
                                          'vision_tsv_to_iris=vision_datasets.commands.converter_tsv_to_iris:main',
                                          'vision_tsv_to_coco=vision_datasets.commands.converter_tsv_to_coco:main',
                                          'vision_yolo_to_iris=vision_datasets.commands.converter_yolo_darknet_to_iris:main',
-                                         'vision_gen_metafile=vision_datasets.commands.generate_image_meta_info:main']
```

Comment:
As discussed in our previous meeting, if we consume the data from DatasetHub, we dont have to care about whether the underlying data is iris or coco format